src/backend/discovery/follow.lambda.ts

```diff
@@ -37,13 +37,13 @@ export async function handler(event: ScheduledEvent, context: Context) {
 
   const denyList = await DenyListClient.newClient();
 
-  const initialMarker = await loadLastTransactionMarker(1_800_000 /* @aws-cdk/cdk initial release was at 1_846_709 */);
+  const marker = await loadLastTransactionMarker(1_800_000 /* @aws-cdk/cdk initial release was at 1_846_709 */);
```

Comment:
We probably can change the initial marker to this one (`cdk-constants`'s latest release, which is from December 30, 2019).

```suggestion
  const marker = await loadLastTransactionMarker(4_401_405 /* around December 30, 2019 */);
```